src/services/api.js

```diff
@@ -77,3 +77,13 @@ export function getGist(gist) {
       });
   });
 }
+
+export function version(serverUrl) {
+  return fetch(apiUrl(`/version?server_url=${serverUrl || ''}`))
```

Comment:
the `server_url` is sent as a `POST` payload when doing a `parseRequest`. For the sake of homogeneity, we could do the same here, and it will simplify the `server.go` part as well.

Reply:
I would prefer to keep it as GET. Even we don't build restful service, still in general it's better use correct http methods. In this case we for sure `get` version, not `post`.

Reply:
For `parseRequest` it is sent a payload where the `server_url` means "it should be parsed by that server".
In this `versionRequest` the payload can be understood the same way: "¿which server version would handle that request?"
If we do so, the communication between front and proxy is more homogeneous, and since we're not doing REST, we don't hide information as well.

Reply:
usually the rule of thumb is if your resource is verb: use POST.
if your resource is noun use appropriate http method.
In our case `parse` is verb, that is why it post and it's fine.
But `version` is noun.

I'm strongly disagree about messing up with http methods, but whatever. Changed.